date_detection/date_column_type_detection.py

```diff
@@ -0,0 +1,144 @@
+"""
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+"""
+This module will detect the type(Consistent, all ambiguous, inconsistent column)
+a passed column is & also wether day_first is True/False for that column.
+"""
+import datetime
+import pandas
+
+def get_column_type_and_order(column):
+    """
+    This function classifies the column into the 3 types based on
+    the values in it. It also returns the day_first = Bool for the column.
+
+    Args - column - type list
+    Returns - dict - {type_str, day_first}}
+                type_str = 'Consistent' / 'all ambiguous' / 'Inconsistent'
```

Comment:
Can we use an enum here instead of strings?

Reply:
Alternatively, the strings should be like enums eg: 'CONSINTENT', 'ALL_AMBIGUOUS', 'INCONSISTENT'

Reply:
Done